ironfish/src/wallet/walletdb/accountValue.ts

```diff
@@ -25,6 +26,7 @@ export class AccountValueEncoding implements IDatabaseEncoding<AccountValue> {
     bw.writeBytes(Buffer.from(value.incomingViewKey, 'hex'))
     bw.writeBytes(Buffer.from(value.outgoingViewKey, 'hex'))
     bw.writeBytes(Buffer.from(value.publicAddress, 'hex'))
+    bw.writeU8(value.version)
```

Comment:
256 versions seems safe in terms of upper limit, but I think 16bit is even safer

Reply:
Don't you need version to be 1st to descriminate deserialization on it?

Reply:
@jowparks sounds good!
@NullSoldier version last is doable but insanely janky compared to version first, combined with your "let's just do a migration" comment I'll happily move it to the front